### ISSUE TITLE
Feedback/suggestions on explainer.md

### ISSUE THREAD

**Issue by coinop-logan** [deathtothecorporation/vaporware.network#16]
@thelifeandtimes lmk if there's a better way to do this in the future!

https://github.com/deathtothecorporation/vaporware.network/blob/3394a67b1849d02088f723cbea8b8878c967804b/explainer.md?plain=1#L12

Replace `better user -- or shall we say, *operator* -- experience` with `better user experience -- or shall we say, *operator* experience`. Easier to mentally parse.

https://github.com/deathtothecorporation/vaporware.network/blob/3394a67b1849d02088f723cbea8b8878c967804b/explainer.md?plain=1#L27

Not sure what you mean by "namespacing", and a search for the term finds nothing else in the doc.

https://github.com/deathtothecorporation/vaporware.network/blob/3394a67b1849d02088f723cbea8b8878c967804b/explainer.md?plain=1#L57

This sentence is hard to parse and seems somehow malformed: "This enables a mechanism for taking the ‘programmability’ of crypto asset ownership, to use as a licensing layer for cloud software, linked to on-chain identity."

https://github.com/deathtothecorporation/vaporware.network/blob/3394a67b1849d02088f723cbea8b8878c967804b/explainer.md?plain=1#L67

This is the first time "mOS" is used as its own term. Somehow I feel like you should put quotes around it, so that the reader doesn't feel like he should already know what this means.

https://github.com/deathtothecorporation/vaporware.network/blob/3394a67b1849d02088f723cbea8b8878c967804b/explainer.md?plain=1#L72

I think that "Each token in the system" should be more specific: "Each mOS pill" or "Each mOS NFT token"...

https://github.com/deathtothecorporation/vaporware.network/blob/3394a67b1849d02088f723cbea8b8878c967804b/explainer.md?plain=1#L101

"Vaporware" -> "vaporware" (right? I think?)

https://github.com/deathtothecorporation/vaporware.network/blob/3394a67b1849d02088f723cbea8b8878c967804b/explainer.md?plain=1#L128

"get's" -> "gets"

https://github.com/deathtothecorporation/vaporware.network/blob/3394a67b1849d02088f723cbea8b8878c967804b/explainer.md?plain=1#L155

I think these two paragraphs can drive the point home much more clearly. Giving it a shot here:

But here is where things get interesting… For Vaporware Token Gated Accounts we have added a specific functionality: The ability to "bond" your in-app, mobile wallet to the top level NFT - in the case of MiladyOS, the Milady Maker token.

This "bonded address" can perform certain on-chain actions within the MiladyOS system on behalf of your Milady - while the Milady itself never moves from where the user had it originally (Metamask, ledger, whatever). This bond can be removed or modified by the Milady holder at any point, and expires if the Milady moves. In the end, this means that the security of the underlying asset is not impacted, and yet the mobile app is released from the awkward flow of signing transactions outside the app. Want to buy a new hat and equip it? You can do this right from your phone, in four taps, without leaving the app.

https://github.com/deathtothecorporation/vaporware.network/blob/3394a67b1849d02088f723cbea8b8878c967804b/explainer.md?plain=1#L260

No idea what it might mean to have code be physically portable.

**Comment by thelifeandtimes** [comment]
not sure why this didn't auto close when @vcavallo merged the recent PR... closing now.
